### ISSUE TITLE
id should be an int

### ISSUE THREAD

**Issue by juloaiza** [geometalab/geojson2vt#6]
Why the id is converted to [str](https://github.com/geometalab/geojson2vt/blob/master/geojson2vt/feature.py) (line 3)? 

Based on vector tile spec, it should be and [uint64](https://github.com/mapbox/vector-tile-spec/blob/master/2.1/vector_tile.proto) line 32

**Comment by lucasfujita** [comment]
I agree. I'm using the geojson2vt lib with [vt2pbf](https://github.com/DenysMoskalenko/vt2pbf) and I need to manually cast id's to int so it don't raise me an error
